heartbeat/aws-vpc-move-ip

```diff
@@ -213,25 +224,35 @@ ec2ip_validate() {
 }
 
 ec2ip_monitor() {
-        MON_RES=""
+	MON_RES=""
+	if ocf_is_true ${OCF_RESKEY_eni_id_lookup}; then
+		EC2_ATTRIBUTE_LOOKUP="NetworkInterfaceId"
+    	EC2_ID="$(ec2ip_get_instance_eni)"
```

Comment:
Let's use tabs instead of spaces. I strongly prefer four spaces myself, but it's best to stay consistent with the rest of the script that uses tabs.

Reply:
He already fixed that.

Reply:
No.
```
$ git clone git@github.com:schubergphilis/resource-agents.git
$ cd resource-agents
$ grep -n '^ ' heartbeat/aws-vpc-move-ip 
230:    	EC2_ID="$(ec2ip_get_instance_eni)"
231:    	identification="interface"
232:  	else
233:    	EC2_ATTRIBUTE_LOOKUP="InstanceId"
234:    	EC2_ID="$EC2_INSTANCE_ID"
235:    	identification="instance"
236:  	fi
328:  	EC2_NETWORK_INTERFACE_ID="$(ec2ip_get_instance_eni)"
```